st2common/st2common/constants/system.py

```diff
@@ -30,3 +30,11 @@
 
 # Used to prefix all system variables stored in the key-value store.
 SYSTEM_KV_PREFIX = 'system'
+
+# Used to prefix all user scoped variables in key-value store.
+USER_KV_PREFIX = 'user'
+
+ALLOWED_KV_PREFIXES = [
+    SYSTEM_KV_PREFIX,
```

Comment:
I think we should think about it some more on how to handle system / global scope.

Right now this is using `SYSTEM_KV_PREFIX` prefix which is totally unrelated to scopes and used for a different purpose. We only use this feature so we can identify, that inside a Jinja variable expression, user wants to reference value from a datastore.

So we should think if it makes sense to conflate / combine those things together or it will just make it more confusing.